terraform/shared/shared.tf.json.template.py

```diff
@@ -45,7 +45,10 @@ def conformance_pack(name: str) -> str:
     CloudTrailAlarm(name='api_unauthorized',
                     statistic='Sum',
                     filter_pattern='{($.errorCode="*UnauthorizedOperation") || ($.errorCode="AccessDenied*")}',
-                    threshold=12,
+                    # We expect one false alarm caused by AWS Config every day
+                    # (see delivery_frequency below) due to a known issue
+                    # regarding Macie/Security hub resource scanning.
+                    threshold=1,
```

Comment:
```suggestion
                    threshold=0,
```

https://github.com/DataBiosphere/azul/issues/6736#issuecomment-2942591993